pom.xml

```diff
@@ -69,24 +69,25 @@
         <!-- versions -->
 
         <io.monix.version>3.1.0</io.monix.version>
-        <kafka-clients.version>2.3.0</kafka-clients.version>
-        <embedded-kafka>2.3.0</embedded-kafka>
-        <json4s-native.version>3.6.0</json4s-native.version>
-        <json4s-jackson.version>3.6.1</json4s-jackson.version>
-        <json4s-ext>3.6.0</json4s-ext>
+        <kafka-clients.version>3.3.2</kafka-clients.version>
```

Comment:
Sorry, I missed this part.
This version might not work with the Kafka version on our cluster.
We need to check the compatibility

Reply:
I checked the compatibility.
According to the bidirectional client compatibility, we can use the Kafka client.
> Kafka has a "bidirectional" client compatibility policy. In other words, new clients can talk to old servers, and old clients can talk to new servers. This allows users to upgrade either clients or servers without experiencing any downtime.
https://kafka.apache.org/protocol#protocol_compatibility